editor/src/core/shared/element-template.ts

```diff
@@ -2018,6 +2018,11 @@ export function elementInstanceMetadata(
 export type DetectedLayoutSystem = 'flex' | 'grid' | 'flow' | 'none'
 export type SettableLayoutSystem = 'flex' | 'flow' | 'grid' | LayoutSystem
 export type TextDirection = 'ltr' | 'rtl'
+export type HugProperty = 'hug' | 'squeeze' | 'collapsed'
```

Comment:
I wonder if we should explicitly have a 'not-hugging' here instead of null doing double-duty